src/open_inwoner/cms/collaborate/urls.py

```diff
@@ -6,20 +6,32 @@
     PlanActionEditStatusTagView,
     PlanActionEditView,
     PlanActionHistoryView,
-    PlanCreateView,
+    PlanCreateNoTemplateView,
+    PlanCreateWithTemplateView,
     PlanDetailView,
     PlanEditView,
     PlanExportView,
     PlanFileUploadView,
     PlanGoalEditView,
     PlanListView,
+    PlanTemplateChooseView,
 )
 
 app_name = "collaborate"
 
 urlpatterns = [
     path("", PlanListView.as_view(), name="plan_list"),
-    path("create/", PlanCreateView.as_view(), name="plan_create"),
+    path(
+        "choose-template/",
+        PlanTemplateChooseView.as_view(),
+        name="plan_choose_template",
+    ),
+    path("create/", PlanCreateNoTemplateView.as_view(), name="plan_create_no_template"),
+    path(
+        "create-from-template/",
+        PlanCreateWithTemplateView.as_view(),
+        name="plan_create_with_template",
+    ),
```

Comment:
- `PlanCreateWithTemplateView` should be `PlanCreateFromTemplateView` imho
- This will likely need a parameter for the template, which is determined on the template choice page, hence:
```
path(
    "create-from-template/<int:template_id>/",
    PlanCreateFromTemplateView.as_view(),
    name="plan_create_with_template",
)
```
However, this suggestion depends on splitting up the form for creating plans into (a) a form for creating without templates and (b) a form for creating with templates. See general comment.